src/main/java/org/jitsi/jigasi/transcription/LocalJsonTranscriptHandler.java

```diff
@@ -218,7 +216,7 @@ public void publish(ChatRoom room, TranscriptionResult result)
     protected void createEncapsulatingObject(JSONObject encapsulatingObject,
```

Comment:
is this method still used?

Reply:
Its used in RemoteTranscriptionPublisher. Not sure, if that should be changed.

Reply:
Yes, you can update it there